### DISCUSSION TITLE
dual battery

### DISCUSSION THREAD

**Discussion by doniprdna** [MultimediaLucario/Lenovo-ThinkPad-T440s#34] [Q&A]
hai, how to make dual battery work?

Model: Thinkpad T470
Bios: 1.72
CPU: Intel(R) Core(TM) i5-6300U CPU @ 2.40GHz
GPU: Intel HD Graphics 520
Storage: NVMe 512GB
RAM: 16GB
Screen: 14" FHD (1920x1080)
Wi-Fi: Intel Dual Band Wireless AC 8260 with bluetooth 
Ethernet: Intel I219-LM 
Sound: Realtek ALC298
oc 0.8.8
macos big sur

**Comment by MultimediaLucario** [comment]
This is the T440s repo. The repo for the T470 can be found here: https://github.com/MultimediaLucario/Lenovo-ThinkPad-T470
